hack/update-codegen.sh

```diff
@@ -18,6 +18,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+set -x
```

Comment:
is that intentional to be kept (it's going to be more verbose)

Reply:
right, I should remove it, but I wanted to make sure it passes before :sweat_smile: